tasks/configure.yml

```diff
@@ -127,6 +127,19 @@
   with_items: "{{ haproxy_listen }}"
   when: haproxy_listen is defined
 
+## ASSEMBLE CONFIG - log-forward
+- name: Gather the package facts
+  package_facts:
+    manager: auto
```

Comment:
This operation has a serious performance impact, it might be preferable to remove this task and let the user judge of its haproxy version.

Reply:
@tleguern : 
Version check have been moved to dedicated task file (tasks/configure-logForward.yml) and will only be performed if the log-forward section is present.
This check is a must to make sur that haproxy version is 20.3 or above so the playbook could proceed generating the log-forward configuration section. Otherwise, the final merged haproxy configuration could not be appropriate, and restarting of haproxy service may fail.